CONTRIBUTING.md

```diff
@@ -14,6 +14,8 @@ If you need any help, don't hesitate to ask the community on [Gitter](https://gi
 * If you are adding functionality or fixing a bug - add a test! Prefer adding new test cases over modifying existing ones.
 * Make sure there is no trailing spaces in code: `./check_trailing_spaces.sh`.
 * Regenerate README: `php php-cs-fixer readme > README.rst` (Windows `php.exe php-cs-fixer readme > README.rst`). Do not modify `README.rst` manually!
+* Install dev tools: `dev-tools/install.sh`
+* Run static analysis using PHPStan: `php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse`
```

Comment:
i wonder why setting memory limit. i don't on my local machine, we don't on CI

Reply:
I do need it on my Docker setup. Maybe your setup and CI have higher limits by default?